README.md

```diff
@@ -88,29 +88,30 @@ Here's the generated DOM tree, as viewed in Chrome inspector:
 A couple of things to note from the code:
 
 1. There are no framework classes to inherit and extend from.
-2. There are no API method calls.
-3. There are no HTML body tags.
-4. All we have is a single JSON-like variable.
-5. The DOM just builds itself without you running any function.
+2. There are no API method calls. The only API is a set of six `$` keywords.
+3. HTML body tags not needed. Drop a script right in `<html>` or give your Genotype the same `id` as an existing div.
```

Comment:
"Genotype" is confusing. It either needs to be clearly defined before using it, or (I recommend) not used at all in the README.